group.html

```diff
@@ -88,6 +88,20 @@ <h3>Ryan Morris
 												<li><a href="http://ryanramorris.space" class="button">Website</a></li>
 											</ul>
 										</article>
+										<article>
+											<a href="#" class="image"><img src="images/aj_photo_1.jpg.jpg" alt="" /></a>
+											<h3>Andrew Jolly
+											<a href="https://twitter.com/iamstillandrew" class="icon brands fa-twitter"><span class="label">Twitter</span></a>
+											<a href="https://github.com/andrewjjolly" class="icon brands fa-github"><span class="label">Github</span></a>
+											<a href="mailto:andrew.j.jolly@unsw.edu.au" class="icon fa-envelope"><span class="label">Email</span></a>
+											</h3>
+											<p>I am a first year physics PhD student at UNSW in Sydney. I am interested in utilizing machine learning techniques to
+												build accurate models of stellar surfaces and then use those models to find exoplanets in radial velocity observations.

```

Comment:
To be pedantic: you're going to find the exoplanets in radial veocity datasets, the observations are the moment of collecting the photons at the telescope.

Reply:
Ah yes. Not that pedantic to be fair.